Automattic/wp-calypso
# Move .checkout-container styles (aka cart on the right) to the checkout section

Removes the styles from the global `client/layout` module and distributes them to better places: a new `checkout-container.scss` file, and existing `cart/style.scss`.

These styles were added by @niranjan-uma-shankar in #33560 (move cart to the right). Relevant to the `Layout` style migration by @diegohaz in #35583.

**How to test:**
Verify that the shopping cart during checkout and payment (in right sidebar) is correctly styled.
